modules/signals/project.json

```diff
@@ -45,15 +47,8 @@
       "outputs": ["{options.outputFile}"]
     },
     "test": {
-      "executor": "@nx/jest:jest",
-      "options": {
-        "jestConfig": "modules/signals/jest.config.ts",
-        "runInBand": true,
-        "passWithNoTests": false
-      },
-      "outputs": ["{workspaceRoot}/coverage/modules/signals"]
```

Comment:
should we keep the coverage path?

Reply:
That's true for the pipeline, but we can still generate reports locally to check the coverage. Since we have the same coverage path defined for other projects within the repo (`coverage` dir is also added in `.gitignore`), I suggest adding it back for consistency.

Reply:
True. Added back